api/envoy/config/metrics/v3/metrics_service.proto

```diff
@@ -21,6 +21,23 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Metrics Service is configured as a built-in *envoy.stat_sinks.metrics_service* :ref:`StatsSink
 // <envoy_v3_api_msg_config.metrics.v3.StatsSink>`. This opaque configuration will be used to create
 // Metrics Service.
+//
+// For example, with the following configurations (in YAML):
+//
+// MetricsService Config
+//
+// .. code::
+//
+//   ...
+//  stats_sinks:
+//    - name: envoy.stat_sinks.metrics_service
+//      typed_config:
+//        "@type": type.googleapis.com/envoy.config.metrics.v3.MetricsServiceConfig
+//        transport_api_version: V3
+//        grpc_service:
```

Comment:
Would it make sense to include other parameters in the example too?

Reply:
Like `address`, `port` of `grpc_service `?

Reply:
What I want to show here is a complete example, But after thinking about it, I decided to remove the other parameters and keep only `metric service config`